### ISSUE TITLE
3.0.0 release plan

### ISSUE THREAD

**Issue by tk0miya** [sphinx-doc/sphinx#7124]
According to annual release cycle, this April is time for major release.
I think this schedule is good for the release. What do you think?

* early March: call for translation
* early or mid of March: beta release (feature freeze)
* early April: final release

In addition, we need to make a new branch for managing 3.x series. I just post a proposal issue #7121. I'd like to make it after discussion ended (as soon as possible). Please give a comment for branch.

Note: issues marked as 3.0.0: https://github.com/sphinx-doc/sphinx/milestone/33

**Comment by tkoyama010** [comment]
>Note: issues marked as 3.0.0: https://github.com/sphinx-doc/sphinx/milestone/33

@tk0miya Maybe https://github.com/sphinx-doc/sphinx/milestone/58 .

**Comment by tk0miya** [comment]
@tkoyama010 That's right. It is copy-pasted :-p

**Comment by tk0miya** [comment]
>early March: call for translation

Today, I post a call for translation for sphinx-users and sphinx-dev group.

**Comment by tk0miya** [comment]
> early or mid of March: beta release (feature freeze)

I'll bump a first beta release and call feature freeze in next weekend. Are you ready?

**Comment by tkoyama010** [comment]
I'm ready.

**Comment by jakobandersen** [comment]
I have some major changes to the C domain that would be nice to merge into 3.x. I plan on a PR within a day or two.

**Comment by tk0miya** [comment]
I also have some discussions still in progress (#7068 and #7301). It would be better to postpone a feature freeze for a week. We're not in a hurry.

**Comment by tk0miya** [comment]
In Japan, there is a hack-a-thon event for documentation. I'll work for this release. So I'll call feature freeze in a day.

I'll continue to work for #7068 and #7031. But their merge might be postponed to 3.0.0b2 or 3.1.0.

**Comment by tk0miya** [comment]
It seems almost issues and PRs on the 3.0 milestone are closed or merged. So I'm going to build a beta package from now on.

**Comment by tk0miya** [comment]
Now I just released Sphinx-3.0.0b1!!
https://pypi.org/project/Sphinx/3.0.0b1/

And I opened a branch `3.0.x` for maintenance 3.0.0 final and following bug fix releases.

* `master`: mainly for 4.0.x release. All changes including breaking one are allowed.
* `3.x`: for 3.1.0 release. Only stable changes are allowed.
* `3.0.x`: for 3.0.0 release. Only bug fixes are allowed.

Thank you for your effort, always.

Best regards,
